src/models/countries.js

```diff
@@ -0,0 +1,46 @@
+module.exports = (sequelize, DataTypes) => {
+  const Country = sequelize.define("Country", {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      allowNull: false,
+      primaryKey: true,
+    },
+    nameOfCountry: {
```

Comment:
have an enum data type and add all the countries of Africa in an array

Reply:
corrected